Tensile/Components/LraTileAssignment.py

```diff
@@ -130,60 +130,6 @@ def __call__(self, writer, kernel, tP):
             "0. thread id in wave: wtid = tid %% wavelength(%u)" % waveWidth)
         kStr += vectorStaticRemainder(dummy, tReg, kReg, kernel["MatrixInstN"], tmpVgpr, tmpSgpr, \
             "1. N offset: nIdx = wtid %% MI_N(%u)" % kernel["MatrixInstN"])
-
-        # offset calculation for TLU=1 when glvw * bpe * wavefrontsize > 256
-        if (kernel["DirectToLds%s" % tP["tensorChar"]] and kernel["ProblemType"]["TLU%s" % tP["tensorChar"]] and (kernel["GlobalLoadVectorWidth%c"%tP["tensorChar"]] * tP["bpe"] * kernel["WavefrontSize"] > 256)):
-          # x2/x4 directToLds stores 8/16 bytes into LDS like below
-          # address offset in LDS in bytes
-          # DWORD# written by LDS_DMA
-          #  address offset in LDS (byte offseet)    
-          #  0    4    8    12    16   20   24   28   32   36   40   44    48    52   56   60 
-          #  data dword#:                            
-          #  0    4    8    12    2    6    10   14    1   5    9    13     3    7    11   15
-          #  Noffset calculation for VW =1 (BPe=8) / VW =2 (BPE=4)
-          #  use direcToLds for best VW and GRVW case; other cases requires bit more lane manipulation..
```

Comment:
Is this configuration no longer supported?

Reply:
This is not supposed to be changed.
LDS data layout from global memory is independent from NN or NT.
Just width (dword_x2 or x4) matters.